### PR TITLE
fix(ci): Fix circleCI

### PR DESCRIPTION
**Pull request by fokusferit**
Sadly,

I forgot doing the build step also in Tests.

### CODE REVIEW

**Comment by fokusferit** [comment]
👍

File: gulpfile.js [hunk 1]
```diff
@@ -107,11 +107,11 @@ gulp.task('default', () => gulp.parallel('sass', 'lint-sass'));
 gulp.task('build:sass', () => 
   gulp.src('./index.scss')
   .pipe(sass({outputStyle: 'compressed'}).on('error', sass.logError))
-  .pipe(gulp.dest('./dist'))
+  .pipe(gulp.dest('./build'))
 );
 
 gulp.task('clean:dist', () => 
-  del(['dist/**'])
+  del(['build/**'])
```
Comment (fragsalat) [inline gulpfile.js]: Why the folder changed to build? In components it's dist as well

Comment (fokusferit) [reply gulpfile.js]: I was just unsure if it might be used somewhere that's why I did a new name. Last build failed because .dist is in .gitignore.

Comment (fragsalat) [reply gulpfile.js]: I guess dist is in gitignore by default^^ I would propose to remove it there and have it unified in all repos

**Pull request merged**